backend/core/database.py

```diff
@@ -976,19 +1014,18 @@ def _update_bgp_updates(self):
             # Update the BGP entries using the handled messages
             if self.handled_bgp_entries:
                 try:
-                    cmd_ = 'UPDATE bgp_updates SET handled=true FROM (VALUES %s) AS data (key) WHERE bgp_updates.key=data.key'
-                    psycopg2.extras.execute_values(
-                        self.db_cur,
-                        cmd_,
-                        self.handled_bgp_entries,
-                        page_size=1000
-                    )
-                    self.db_conn.commit()
+                    query = 'UPDATE bgp_updates SET handled=true FROM (VALUES %s) AS data (key) WHERE bgp_updates.key=data.key'
+                    with get_wo_cursor(self.wo_conn) as db_cur:
+                        psycopg2.extras.execute_values(
+                            db_cur,
+                            query,
+                            self.handled_bgp_entries,
+                            page_size=1000
+                        )
                 except Exception:
                     log.exception(
                         'handled bgp entries {}'.format(
                             self.handled_bgp_entries))
-                    self.db_conn.rollback()
```

Comment:
aren't rollbacks not needed anymore?

Reply:
they are inside the context of `get_ro_conn()` and `get_wo_conn()`